### PR TITLE
Us6 title no to deed

### PR DESCRIPTION
**Pull request by Phemus**

### CODE REVIEW

**Comment by tsangsimon** [comment]
column was incorrectly named json_doc on the table and requested this be changed to deed.
rewritten the commit history to remove the unnecessary migration version that dropped the deed table.

**Comment by tsangsimon** [comment]
:+1:

**Comment by Kirosoft** [comment]
:+1:

File: application/deed/views.py [hunk 1]
```diff
@@ -1,11 +1,68 @@
-
+from application.deed.model import Deed
+from flask import request, abort
 from flask import Blueprint
+from flask.ext.api import status
+from jsonschema.validators import validator_for  # type: ignore
```
Comment (Phemus) [inline application/deed/views.py]: type: ignore still needed?

**Comment by Phemus** [comment]
:+1: appart frpm comment above

File: application/deed/views.py (hunk 1 above)
Comment (tsangsimon) [inline application/deed/views.py]: commit c9adefa removes this

**Pull request merged**